### DISCUSSION TITLE
🎉 Quarkus joins Commonhaus!

### DISCUSSION THREAD

**Discussion by aalmiray** [commonhaus/foundation#200] [Announcements]
Windermere, Florida - All remote – Sept 23, 2024 –  Today, the [Commonhaus Foundation](https://www.commonhaus.org/) is pleased to announce that Quarkus joins as a member project.

[Quarkus](https://quarkus.io/) was unveiled as a community project in[ March 2019](https://dandreadis.blogspot.com/2019/05/getting-quarked.html) with the goal of bringing back Developer Joy and revolutionizing Java for the Cloud Era. Five years later, as explained in their initial[ announcement](https://quarkus.io/blog/quarkus-in-a-foundation/), the project has succeeded in its original goals and Quarkus has captivated the developer community with a vibrant and ever-expanding ecosystem of 700+ publicly available extensions (the Quarkus universe or [Quarkiverse](https://github.com/quarkiverse) as they call it),  900+ contributors and close to 1 million monthly downloads of its core artifacts.

[Dimitris Andreadis](https://x.com/dandreadis), a JBoss Veteran & Open Source Advocate who is also the Engineering Director overseeing Quarkus, had this to say: 

> As adoption increases, our vision for the future is to establish Quarkus as the de-facto standard for new Java-based applications, by welcoming more developers and vendors from across the software ecosystem, and inviting them to be part of this dynamic journey. Towards that goal we intend to remove any remaining obstacles, including the perception that Quarkus is too dependent on Red Hat--the company that sponsored the creation of Quarkus and will continue to sponsor it for the foreseeable future--by moving Quarkus to an open Foundation, thus bolstering our commitment to open-source development and inspiring greater community participation.
>
> After a period of consulting with the community inside and outside Red Hat we decided to submit a request for the Quarkus project to [join the Commonhaus Foundation](https://quarkus.io/blog/quarkus-moving-to-commonhaus/). Commonhaus stands out with its innovative governance framework and commitment to independently managed projects. Joining Commonhaus allows Quarkus to maintain its identity and benefit from a self-governing model. We also believe that Commonhaus suits Quarkus' fast and agile delivery model.
>
> We are also excited to be in the good company of the other[ ten prominent projects](https://www.commonhaus.org/#our-projects) that are currently hosted in Commonhaus, which include Hibernate and Jackson, both important components of Quarkus, and we very much encourage others to follow suit.
>
> This move is as much for Quarkus as it is for you. If you or the organization you work at are interested in ensuring that the Quarkus ecosystem grows and evolves in a direction that helps you, come and join us in helping Quarkus transition to the CommonHaus foundation.
>
> As we said when we created Quarkus, join the revolution!

Commonhaus is elated to have Quarkus be among its member projects.

**Comment by JooHyukKim** [comment]
Welcome Quarkus team!
